models/time/src/migration.ts

```diff
@@ -57,7 +58,8 @@ export async function migrateWorkSlots (client: TxOperations): Promise<void> {
             workslots: 0,
             priority: ToDoPriority.NoPriority,
             user: account.person,
-            visibility: 'public'
+            visibility: 'public',
+            rank: '' // TODO
```

Comment:
Migration should calculate them in some proper order, not just ''

Reply:
Done